lib/PDFfiller/index.js

```diff
@@ -0,0 +1,130 @@
+const Api = require('./../Api/');
+const constants = require('../constants/');
+
+let INSTANCE; // discus about it
+
+/**
+ * The JavaScript API to PDFFiller services
+ * @class PDFfillerApi
+ * @param {object} [config]
+ * @param {boolean} [config.autoTokenUpdate=true] set up auto update token using refresh_token
+ */
+
+class PDFfillerApi {
+  constructor(config) {
+    config = config || {};
+    INSTANCE = this;
+    this.api = new Api();
+    this.autoTokenUpdate = config.autoTokenUpdate || true;
+    this.__accessToken = null;
```

Comment:
privates.set(this, {
  accessToken: null,
  api: new Api(),
 autoTokenUpdate: ...
})